src/current/_includes/releases/v23.1/v23.1.13.md

```diff
@@ -0,0 +1,87 @@
+## v23.1.13
+
+Release Date: December 4, 2023
+
+{{ site.data.alerts.callout_info }}
+<release_version> is currently available only on select CockroachDB {{ site.data.products.dedicated }} and {{ site.data.products.serverless }} clusters. If you wish to upgrade your cluster to this version, please submit a [support request](https://support.cockroachlabs.com/hc/en-us/requests/new).
+
+This release is currently not available for CockroachDB {{ site.data.products.core }} customers.
+{{ site.data.alerts.end }}
+
+{% comment %}{% include releases/release-downloads-docker-image.md release=include.release %}{% endcomment %}
+
+<h3 id="v23-1-13-sql-language-changes">SQL language changes</h3>
+
+- A new option for the [`RESTORE`](https://www.cockroachlabs.com/docs/v23.1/restore.html) syntax, `strip_localities`, has been added. This can be used to strip the [locality information](https://www.cockroachlabs.com/docs/v23.1/alter-table.html#set-locality) from a backup when there are mismatched [cluster regions](https://www.cockroachlabs.com/docs/v23.1/multiregion-overview.html#cluster-regions) between the backup's cluster and the target cluster. The following are behaviors that will most likely not be encountered with the specific use case that this patch provides, but are documented nonetheless:      - Adding a [primary region](https://www.cockroachlabs.com/docs/v23.1/alter-database.html#set-primary-region) to a regionless restore (with or without [regional by row table(s)](https://www.cockroachlabs.com/docs/v23.1/table-localities.html#regional-by-row-tables)) will not work out-of-the-box, but does produce an accurate message detailing a user to essentially [`DROP TYPE [database].public.crdb_internal_region;`](https://www.cockroachlabs.com/docs/v23.1/drop-type.html) (and for cluster restores, [`ALTER DATABASE [database] CONFIGURE ZONE DISCARD;`](https://www.cockroachlabs.com/docs/v23.1/alter-database.html#remove-a-replication-zone)).
+    - Restoring a cluster/database/table with a regional by row table will not work out-of-the box. In particular, if we are performing writes, the [`crdb_region` column](https://www.cockroachlabs.com/docs/v23.1/alter-table.html#crdb_region) needs to specify the region of the new row(s) being written to the table. The user will need to alter said column and set a default that makes sense, along with discarding the zone config (this latter is due to the fact that the zone config holds all outdated info related to the [partitions](https://www.cockroachlabs.com/docs/v23.1/partitioning.html), [constraints](https://www.cockroachlabs.com/docs/v23.1/constraints.html), etc.). These are due to a conflict with the `crdb_region` column already being present in the regionless restore. This column specifies each row's home region and is a prefix to the table's primary key. Stripping localities does not touch this column as it would be an expensive operation that includes rewriting the entire table. [#111863][#111863]
```

Comment:
```suggestion
    - Restoring a cluster/database/table with a regional by row table will not work out-of-the box. In particular, when performing writes, the [`crdb_region` column](https://www.cockroachlabs.com/docs/v23.1/alter-table.html#crdb_region) needs to be set to the region of each new row being written to a given table. An administrator should set a default `crdb_region` value for the cluster. To write to a different region, the user must alter the `crdb_region` value, and must also discard the zone config (because the zone config may hold outdated information about [partitions](https://www.cockroachlabs.com/docs/v23.1/partitioning.html), [constraints](https://www.cockroachlabs.com/docs/v23.1/constraints.html), or other cluster metadata). For more details, refer to the pull request. [#111863][#111863]
```